### PR TITLE
Interface changed to abstract class to compress duplicated code. Basi…

### PR DESCRIPTION
**Pull request by Arkronzxc**
…c methods were overridden in TaskRepository.

### CODE REVIEW

File: src/main/java/com/github/arkronzxc/noteserver/repository/CommonRepository.java [hunk 1]
```diff
@@ -1,14 +1,40 @@
...
+        };
+        collection.find().forEach(foreach);
+        return gson.toJson(documents);
+    }
+
+    public abstract String insert(Request req, Response res);
+
+    public abstract String get(Request req, Response res);
+
+    public abstract String delete(Request req, Response res);
```
Comment (senyasdt) [inline src/main/java/com/github/arkronzxc/noteserver/repository/CommonRepository.java]: May be add some logic to abstarct class. Thhiking about delete and get method

**Review by senyasdt — CHANGES_REQUESTED** [review]
Little logic fix

**Review by senyasdt — APPROVED** [review]
Review done

**Comment by Arkronzxc** [comment]
Tasks #7 #8 #9 #13 are done.

**Pull request merged**